cfg/rke2-cis-1.24/master.yaml

```diff
@@ -154,6 +154,7 @@ groups:
         tests:
           test_items:
             - flag: "root:root"
+        type: manual
```

Comment:
From the provided link, the check is labeled "manual." What’s the rationale behind setting the type to "manual"? 

We shouldn't classify checks titled "manual" as type "manual," as they have different meanings.

The "manual" type indicates tests that cannot be automated. These tests always yield a WARN state and require user intervention.

CIS’s tool, CAT, has shifted from "score/not scored" to "Automated/Manual" in the 1.6 CIS benchmark to align with its functionality. We follow CIS benchmarks, which is why we use these titles.

Reply:
@mjshastha That is precisely why we want to keep type as manual. Someone using Kube-Bench recently complained about false positives regarding checks that CIS mentioned as Manual. The issue was that these users were seeing a PASS?FAIL fo rthis check and wanted to cross-check. Setting type as manual will mark this check as WARN and not lead to someone thinking that Kube-Bench has incorrectly marked an audit as PAss/FAIL. WARN will, in some sense, force them to run the audit manually to verify - which is the very purpose of a Manual test to begin with ;)

Reply:
If users have previously complained about false positives, it would be more beneficial to verify the check and adjust its implementation accordingly, rather than marking it as type: manual.

Marking it as manual does not address the root cause of the issue and may lead to confusion about the check's validity. We should only set the type to manual if it is truly a non-automatable check.